src/Simulation/CsharpGeneration/SimulationCode.fs

```diff
@@ -385,7 +334,7 @@ module SimulationCode =
             match ex.Expression with
 // TODO: Diagnostics
             | InvalidExpr                   -> failwith "Can't generate code for error expression"
-            | UnitValue                     -> (``ident`` "QVoid") <|.|> (``ident`` "Instance")
```

Comment:
I like keeping the ticks because it's easier to identify what are calls that are generating C# code.